pkg/extension/sumologicextension/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/collector v0.38.0
 	go.uber.org/zap v1.19.1
+	google.golang.org/grpc v1.41.0
```

Comment:
BTW, this is an effect of `make gomod-download-all`

Reply:
That's correct: because of your patch in https://github.com/SumoLogic/sumologic-otel-collector/pull/348 grpc is now a direct (instead of indirect) dependency 👍